libqtile/config.py

```diff
@@ -52,6 +52,7 @@ def __init__(self, modifiers, key, *commands):
         self.modifiers = modifiers
         self.key = key
         self.commands = commands
+        self.kwds = kwds
```

Comment:
This is why keyword only arguments are so nice...

Since you're only using the `**kwds` for `desc`, maybe you can just get that value here.